app/portainer/components/datatables/components/ColumnVisibilityMenu.tsx

```diff
@@ -3,12 +3,12 @@ import { Menu, MenuButton, MenuList } from '@reach/menu-button';
 import { ColumnInstance } from 'react-table';
 
 import { Checkbox } from '@/portainer/components/form-components/Checkbox';
-import type { DockerContainer } from '@/docker/containers/types';
 
 import { useTableContext } from './TableContainer';
 
 interface Props {
-  columns: ColumnInstance<DockerContainer>[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  columns: ColumnInstance<any>[];
```

Comment:
see what we do in other components, get the type as a generic.

don't disable this error

Reply:
couldn't find an example for this, you mean to use something like the definition in react-table-config?
`ColumnInstance<D extends Record<string, unknown> = Record<string, unknown>>`

Problem is that here ColumnVisibilityMenu makes use of a defined interface and needa a type to be passed

Reply:
I think I now got what you meant, changed it to
`
interface Props<D extends object> {
  columns: ColumnInstance<D>[];
  onChange: (value: string[]) => void;
  value: string[];
}
`